src/main/java/core/basesyntax/operationstrategy/operation/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operationstrategy.operation;
+
+import core.basesyntax.db.StorageDao;
+import core.basesyntax.db.StorageDaoImpl;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    private final StorageDao storageDao = new StorageDaoImpl();

```

Comment:
I don't know that you need ti provide Dao, because in each handler you create a separate entity of StorageDao, so it maybe more optimal to access the Storage directly. But up to you

Reply:
we use class Storage here as a fake DB, but if it was a real DB, we could do that, so from my point, we need DAO